names.csv

```diff
@@ -1144,3 +1144,96 @@ zack,zach,zak
 zebedee,zeb
 zedediah,dyer,zed,diah
 zephaniah,zeph
+adam,adan 
+alberto,beto
+alejandro,alex,alexander
+alfredo,fredi,alfred
+andres,andrew
+anibal,hannibal
+antonio,tono,tony,anthony
+armando,nando,herman
+arturo, arthur
+bernardo,berna,bernard
+candelario,candel
+carlos,charles
+cristian,cris,christian
+cristobal,cris,christopher
+delfino,pillo
+domingo,mingo
+eduardo,lalo,edward
+enrique,quique,henry
+ernesto,neto,ernest
+esteban,steven
+federico,quiquo,kiko,federick
+fernando,nando,ferdinand
+francisco,paco,pancho,francis

```

Comment:
I have a dog named "Paco". I didn't realize that was a nickname for Francisco. I'll have to start calling him that. :)